models/articles.model.js

```diff
@@ -14,3 +14,28 @@ exports.selectArticleById = articleId => {
       }
     });
 };
+
+exports.updateArticleVotes = (article_id, votes) => {
+  if (votes === undefined) {
+    return Promise.reject({
+      status: 400,
+      msg: "This is a bad request - no data provided for patch request"
+    });
+  }
+
+  return db
+    .query(
+      `UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *`,
+      [votes, article_id]
+    )
+    .then(({ rows: article }) => {
```

Comment:
Strictly speaking this recasting isn't quite right as we are assigning the variable name of article to the array that contains the article at the zeroth index. You could make use of nested destructuring here to assign that index to the variable of article.